apps/arweave/src/ar_mining_worker.erl

```diff
@@ -993,15 +993,15 @@ mark_single_chunk2_missing_or_drop(Nonce, NoncesLeft, Candidate, State) ->
 %% @doc Mark the chunk2 as missing for the whole recall range.
 mark_second_recall_range_missing(Candidate, State) ->
 	#mining_candidate{ packing_difficulty = PackingDifficulty } = Candidate,
-	mark_second_recall_range_missing(0, ar_block:get_max_nonce(PackingDifficulty), Candidate, State).
+	mark_second_recall_range_missing(0, ar_block:get_nonces_per_recall_range(PackingDifficulty), Candidate, State).
```

Comment:
interesting - was this the change that was causing an occasional sub-chunk to leak?

Reply:
Yes, this is the think I was sure I fixed previously. If you remember, there was off-by-one error, and I'm pretty sure I did apply the fix for it. After that I didn't even think of the bug being the same thing once again.